examples/systemd/sv2-translator.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Stratum v2 Translator
+After=sv2-translator.service
+
+[Service]
+# Assumes a user stratum with Rust installed and the repository cloned into ~/src
+WorkingDirectory=/home/stratum/src/stratum/roles/translator
+ExecStart=/home/stratum/.cargo/bin/cargo run -p translator
```

Comment:
I assume there's a way to install the binary and avoid `cargo run`?

Reply:
Updated to us `cargo build -p translator --release` and point to the target dir.

It would be nice if the working directory could just be some random temp directory, but it won't start when I do that.

Reply:
$USER/src/stratum/target/release/translator 

?

Reply:
`/usr/local/bin` is not accesible for my `stratum` user (no sudo rights).

`~/.cargo/bin` would be a better location, but I couldn't get the something like `cargo install` to work. Manually copying the binary feels like a hack, but it's an option.

Reply:
Not sure if $USER works inside a systemd service file?